lib/sla.js

```diff
@@ -116,6 +116,8 @@ function originateChannel(client, inbound, bridge, trunkData) {
   function dialedEnteredStasis(event, line) {
     var answer = Q.denodeify(line.answer.bind(line));
     answer();
+    var deviceState = Q.denodeify(client.deviceStates.update.bind(client));
```

Comment:
From a readability perspective, put some whitespace between logical groupings of code:

```
var answer = ...
answer()

var deviceState = ...
deviceState(...)

var toKill = ...
```

etc.